### ISSUE TITLE
(Bulk) Renaming using placeholders for table column values

### ISSUE THREAD

**Issue by Wolfsblvt** [Wolfsblvt/BlossomsPokemonGoManager#47]
I want to add an idea to the "renaming" feature.

It would be really cool if you can define how the name is set together yourself. I'll give an example. So the simplest thing would be to just use IV percentage, like `magicarp_97`. But for me, I really like to have the starting CP in the name, so that I can consider after I have evolved it if another pokemon would save me some dust if the IV is similar, cause it's CP is already higher. My current naming scheme is `590; 84.4`.

Okay, long talk, short thing. Every person likes different names. How about generating the name from placeholders pointing at the table columns? So I would specify `%cp%; %iv_percent%` for my scheme. So it is also possible to use %species%, %id%, and so on.

Maybe it's a bit... overweighted my idea, but I would really love to see that.

**TL;DR:** Suggestion: Make renaming using placeholders for table column values.

**Comment by JaymerJaymer** [comment]
yes, ok...

to help me, i put a 1 digit code in front (since their sorting ingame is lacking) so it groups mine by Type.

g eggs   (g is grass)
w seel  (w is water)

actually, i do 
w ice 45 so when i'm sorting by Name in my ingame list, i can get to all the Water easily, then see what their big attack does.   Now I can use that google DUEL recap sheet to know the best attach over 100 seconds and I'll put this after my 1 digit category.

**Comment by Wolfsblvt** [comment]
@wullxz wanted to do that.

**Comment by Wolfsblvt** [comment]
In #71 was mentioned that inline edit of nicknames would be a really cool feature. I agree. We should add that with the same PR that includes this stuff here. It sounds like it would be fine with the same internal function.
Pinging @ISeeTanamon, that's the issue for you now.

**Comment by Wolfsblvt** [comment]
Was fixed in #89.
